api/src/pcapi/routes/backoffice/templates/components/links.html

```diff
@@ -60,6 +60,11 @@
      class="link-primary"
      title="Page de l'offre">{{ offer.id }}</a>
 {% endmacro %}
+{% macro build_collective_offer_template_details_link(offer) %}
+  <a href="{{ url_for('backoffice_web.collective_offer_template.list_collective_offer_templates') }}" {# !TODO: need to create function in blueprint in PC-26057 #}
+    class="link-primary"
+  target="_blank">{{ offer.name }}</a>
```

Comment:
`target="_blank"` sert à ouvrir dans un nouvel onglet, ce qu'on ne force que pour les liens externes au BO, donc les liens vers PC Pro ou vers l'annuaire des entreprises.
Ici on n'en a donc pas besoin.
Par contre, comme au-dessus, tu peux mettre un _tooltip_.
```suggestion
  title="Page de l'offre">{{ offer.name }}</a>
```
Il suffisait de copier coller ;)